intgtest/intgSpec.js

```diff
@@ -44,6 +44,10 @@ describe('Vantiq SDK Integration Tests', function() {
         return v.session.accessToken = AUTHTOKEN;
     });
 
+    after(function() {
+        v.unsubscribeAll()
```

Comment:
The websocket was causing the tests to hang after completing, requiring the `--exit` option in the test command. `unsubscribeAll()` closes the websocket along with its subscriptions.